doc/introduction.rst

```diff
@@ -0,0 +1,28 @@
+============
+Introduction
+============
+
+Distance metrics are widely used in the machine learning literature.
+Traditionally, practitioners would choose a standard distance metric
+(Euclidean, City-Block, Cosine, etc.) using a priori knowledge of
+the domain.
+Distance metric learning (or simply, metric learning) is the sub-field of
+machine learning dedicated to automatically constructing optimal distance
+metrics.
+
+This package contains efficient Python implementations of several popular
```

Comment:
emphasize the scikit-learn compatible aspect

Reply:
You're right, done